manta-trusted-setup/src/groth16/ceremony/config/ppot.rs

```diff
@@ -333,18 +318,26 @@ pub fn register(twitter_account: String, email: String) {
         ),
     )
     .expect("Signing message should succeed.");
-    println!(
-        "Your {}: \nCopy the following text to the \"Signature\" Section in Registration Form: \n {}\n",
-        "Signature".italic(),
-        bs58::encode(signature).into_string().green()
-    );
     println!(
         "Your {}: \nThe following text stores your secret for the trusted setup. \
          Save the following text somewhere safe. \n DO NOT share this with anyone else! \
-         Please discard this data after the trusted setup ceremony.\n {}",
+         Please discard this data after the trusted setup ceremony.\n {} \n",
         "Secret".italic(),
         mnemonic.phrase().red().bold(),
     );
+
+    println!("Your registration is not complete until you submit the following form: \nCopy this link to your browser and complete registration there \n \n{}\nBe sure to copy this link exactly! Do not leave out any characters!", register_link(twitter_account, email, bs58::encode(keypair.1).into_string(), bs58::encode(signature).into_string()).green());
+}
+
+/// Generates link to registration form with Twitter, Email, Public Key,
+/// Signature fields pre-filled.
+pub fn register_link(
+    twitter: String,
+    email: String,
+    public_key: String,
+    signature: String,
+) -> String {
+    format!("https://mantanetwork.typeform.com/trustedsetup2#twitter={}&email={}&verifying_key={}&signature={} \n", twitter, email, public_key, signature)
```

Comment:
I tried the form and it still asked me for my email. This is probably a bug right? Also, are we switching everyone to this new form (regardless of what version of the client they are using)? In that case I would just call it `trustedsetup` not `trustedsetup2`.

Reply:
No, Jennifer wanted to keep the email question for reasons to do with integration with Mailchimp. This email address would be for our communications; the one provided by the link is what we would use for signature checking.

We're keeping both typeforms open and active in case some people keep v1 of the client and decide to resubmit registration, or share the v1 `.exe` with a friend, etc. So the old one is called `trustedsetup` and this new typeform is called `trustedsetup2`

Reply:
Why would we have two different emails? It wasn't clear in the UI that they would be different.